src/Smalot/PdfParser/Font.php

```diff
@@ -283,7 +283,9 @@ public function calculateTextWidth(string $text, array &$missing = null): ?float
     {
         $index_map = array_flip($this->table);
         $details = $this->getDetails();
-        $widths = $details['Widths'];
+
+        // TODO: Temporary fix, in the rare case the Widths-key is not set in $details array.
```

Comment:
Why it's temporary? Do you plan to create a better fix?

Reply:
No, but because I was not able to find the origin of the error I wanted to at least mark it for future refactoring.

What would you suggest here?

Reply:
Well regarding the various code of `getDetails()` we can never be sure that a key is defined. So maybe, we should always handle the case where the given key (`Widths` here) is not defined.